packages/core/template/project/AboutUs.md

```diff
@@ -0,0 +1,61 @@
+---
+  layout: default.md
+  title: "About Us"
+---
+
+# About Us
+
+We are a team based in the [Organisation A](#project-team).
+
+You can reach us at the email `organisationa@gmail.sg`
+
+## Project team
+
+### John Doe
+
+{{ johnDoeVariable }}
```

Comment:
On thinking more, this usage of variable doesn't make much sense to me. Essentially you've shown a use of variable that just includes a hardcoded image of a person in global variables - but the usage here seems to imply its some method of formatting ANY image when it only has that one image.

I can think of two alternatives to demonstrate usage of variable
1. Have something "fancy" saved into the variable and use it somewhere else, like IDK some sort of ASCII divider or image that you might want to "reuse" throughout the website. This is probably simplest to show code reuse.
2. The code example in the user guide is that of {{ year }} . This makes sense because you might want to have a "last updated" year or a moving deadline that you update one place which then updates anywhere. If you can think of a good way to represent this I think this would also be interesting.

Reply:
Oh! Or instead of it being `johnDoeVariable` you could call it something like `placeholderImage` . Since you actually use it lots of places besides just here! I think that would be simplest and best.